Library/Formula/strider.rb

```diff
@@ -0,0 +1,87 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
+  end
+end
+
+class Strider < Formula
+  homepage "http://stridercd.com/"
+  url "https://github.com/Strider-CD/strider/archive/v1.6.3.tar.gz"
+  sha1 "6f0fe2b306e2bce395d39b9ce2dc6249bdcf616d"
+
+  depends_on UniversalNpm
+  depends_on "mongodb"
+
+  head "https://github.com/Strider-CD/strider.git", :branch => "develop"
+
+  def install
+    system "npm", "install"
+    prefix.install Dir["bin", "client", "lib", "public", "node_modules", "bower_components"], "main.js", "package.json"
+
+    if File.file?(etc/"striderrc")
+      ohai "Existing config file found at #{HOMEBREW_PREFIX}/etc/striderrc"
+    else
+      (etc/"striderrc").write conf_file
+      ohai "The strider config file has been installed to #{HOMEBREW_PREFIX}/etc/striderrc. Please modify this after the installation."
+    end
+  end
+
+  def caveats
+    "If this is a clean install, you will need to add an admin user. To do so, execute 'strider addUser' and follow the prompts."
+  end
+
+  def conf_file; <<-EOS.undent
+      {
+        "host"        : "0.0.0.0",
+        "port"        : 3000,
+        "server_name" : "http://localhost",
+        "db_uri"      : "mongodb://localhost/strider-foss",
+        "smtp_host" : "",
+        "smtp_port" : 587,
+        "smtp_user" : "",
+        "smtp_pass" : "",
+        "smtp_from" : "Strider <noreply@stridercd.com>",
+        "enablePty"      : false,
+        "extpath"        : "node_modules",
+        "session_secret" : "8L8BudMkqBUqrz",
+        "github_app_id"  : "",
+        "github_secret"  : "",
+        "cors"           : false
+      }
+    EOS
+  end
```

Comment:
Is this provided by upstream somewhere? Would rather avoid hardcoding this if possible.